renderer_elements.js

```diff
@@ -243,6 +261,74 @@ class Node extends SDFGElement {
 }
 
 class Edge extends SDFGElement {
+
+    draw_memory_volume_overlay(renderer, ctx, overlay_man, cutoff_high_volume) {
```

Comment:
I don't think it should be here, but in the overlay manager. Either that or generalize the API from `draw_memory_volume` to `highlight(renderer, ctx, low, hi, color)` or some other general API.

Reply:
I'll vote for putting it into the overlay manager (specifically the memory volume overlay), since a general API that encompasses all further overlays might restrict what we're able to do with future overlays without turning it back on its head